public_officials/tests.py

```diff
@@ -286,3 +287,42 @@ def test_recent_bills_url_resolves_to_bills_index_view(self):
     def test_recent_bills_url_renders_correct_template(self):
         response = self.client.get('/recent-bills/')
         self.assertTemplateUsed(response, 'public-officials/bills/index.html')
+
+class LegislatorRelationshipsTest(TestCase):
+    def test_legislator_has_many_to_many_relationship_with_users(self):
+        legislator = Legislator.objects.create(first_name="Diana",
+                                               last_name="DeGette",
+                                               phone="12345",
+                                               email="email@email.com",
+                                               state_name="Colorado",
+                                               pid="D000197",
+                                               chamber="house",
+                                               term_start="2017-01-03",
+                                               term_end="2019-01-03",
+                                               party="D",
+                                               state="CO",
+                                               cid="N00006134")
+        legislator_two = Legislator.objects.create(first_name="Henry",
+                                                   last_name="Schmojo",
+                                                   phone="12345",
+                                                   email="email@email.com",
+                                                   state_name="Texas",
+                                                   pid="D000197",
+                                                   chamber="house",
+                                                   term_start="2017-01-03",
+                                                   term_end="2019-01-03",
+                                                   party="D",
+                                                   state="TX",
+                                                   cid="N00006134")
+        user = User.objects.create(username='test')
+        user2 = User.objects.create(username='test2')
+
+        legislator.users.add(user)
+        legislator.users.add(user2)
+        legislator_two.users.add(user)
+
+        self.assertEqual(legislator.users.count(), 2)
```

Comment:
Nice tests! We need the tests right now more than ever to help us know what is happening. 👍